### ISSUE TITLE
CanvasTextFormat throws in Unpackaged apps

### ISSUE THREAD

**Issue by mattleibow** [Microsoft/Win2D#941]
This is very similar to https://github.com/microsoft/Win2D/issues/891

But instead of the CanvasFontSet API, I am trying to render an image using text with the `CanvasTextFormat` type. This thows a similar exception: 

`The URI specified in the CanvasTextFormat's FontFamily has an invalid scheme; the scheme may be omitted, or must be one of ms-appx:// or ms-appdata://.`

This is our code: 
https://github.com/dotnet/maui/blob/d8939fdfc48fb36d18a3bb6d9cafb1c09c19117b/src/Core/src/ImageSources/FontImageSourceService/FontImageSourceService.Windows.cs#L55-L95

We are basically rendering a character onto a canvas using some font properties. Fonts work fine in packaged and when I do unpackaged I get an exception. 

I also tried a full path using the `file://` and that does not crash, but it renders the placeholder/missing empty rectangle.

**Comment by mattleibow** [comment]
This issue came up in our case of rendering text to a bitmap: https://github.com/dotnet/maui/issues/15802

**Comment by AathifMahir** [comment]
Facing the same issue here as well:
https://github.com/AathifMahir/MauiIcons/issues/79

**Comment by BurkusCat** [comment]
I have a .NET MAUI app where this issue is causing button icons to break in the unpackaged WinUI version of the app. There doesn't appear to be any workarounds with handlers which is disappointing. The only workaround would be to not use font icons at all and use .svgs which I'm hesitant to do specifically for WinUI unpackaged only.

Would it be possible to have a fix for this issue prioritised, please?

**Comment by jeremy-visionaid** [comment]
Agreed. this should be a priority issue for Win2D as it is a blocker for other downstream issues.

**Comment by AndreasReitberger** [comment]
Facing the same issue. Is there an ETA for this fix from Microsoft?

@mattleibow thanks for open this issue 😊

**Comment by jingliancui** [comment]
Hi team， I met the same issue on Maui, it impacted the unpacked program. Do you have any workaround or fixing plan on this. Thanks so much in advance.

**Comment by AndreasReitberger** [comment]
The issue at the MAUI repository has been closed now. Is the fix from Win2D already read?
@mattleibow 

Thank you!

**Comment by BrundhaVelusamy** [comment]
Is there any update on this?

**Comment by BrundhaVelusamy** [comment]
Any update on this issue?

**Comment by AndreasReitberger** [comment]
@mattleibow 
Any chances that this is checked by Microsoft soon? Thank you!

**Comment by mattleibow** [comment]
Has anyone tried the latest Win2D? All the Maui use cases are now working in 1.2.0

**Comment by jeremy-visionaid** [comment]
It works OK for me since 1.2.0 and I don't see any newer versions

**Comment by AndreasReitberger** [comment]
@SF3969 
Can you verify the fix? It seems that only Syncfusion controls have this issue now.
Maybe an update of the dependencies is needed here as well.

@mattleibow Thank you!

**Comment by AathifMahir** [comment]
> Has anyone tried the latest Win2D? All the Maui use cases are now working in 1.2.0

When can we expect Maui Side to update the Win2D Dependency to v1.2.0? Next Service Release?

**Comment by jingliancui** [comment]
> > Has anyone tried the latest Win2D? All the Maui use cases are now working in 1.2.0
> 
> When can we expect Maui Side to update the Win2D Dependency to v1.2.0? Next Service Release?

Maybe it is better to raise an issue on maui repo

**Comment by mattleibow** [comment]
@AathifMahir what version are you using because this has been fixed for months. Since 8.0.10.

**Comment by AathifMahir** [comment]
> @AathifMahir what version are you using because this has been fixed for months. Since 8.0.10.

This is regarding below issue, more users keeps reporting on this, when it comes v8.0.10 I don't think that fixes the issue

https://github.com/AathifMahir/MauiIcons/issues/79

**Comment by ArunachalamSyncfusion** [comment]
Is there any update on this issue?

**Comment by mattleibow** [comment]
Since we are not able to repro, please attach a repro sample as well as a binlog so we can investigate.

**Comment by SwethavasanthSF3977** [comment]
We have created a repro sample and attached it. Kindly take a look.
[FontFamilyIssue.zip](https://github.com/user-attachments/files/16903817/FontFamilyIssue.zip)

**Comment by BrundhaVelusamy** [comment]
Please provide high priority to resolve this issue.

**Comment by Redth** [comment]
It looks like there's some confusion here between packaged/unpackaged scenarios.

The sample provided by @SwethavasanthSF3977 is using an `ms-appx://` scheme for the font source which won't work in unpackaged app scenarios.

For example, in the sample, the value being used in one case is:  `ms-appx:///Cinzel-Regular.ttf#Cinzel`.  If you instead change this to use `file://` scheme and the full path to the font like this: `file:///C:/code/FontFamilyIssue/bin/Debug/net8.0-windows10.0.19041.0/win10-x64/Cinzel-Regular.ttf#Cinzel` it will work with an unpackaged app.

You can see the PR where MAUI started supporting unpackaged apps by detecting the packaging format and finding the correct path to the file for unpackaged: https://github.com/dotnet/maui/pull/22646
